### PR TITLE
Update dark theme

### PR DESCRIPTION
**Pull request by techexo**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | no
| Documentation | no
| Translation   | no
| CHANGELOG.md  | no
| License       | MIT

Dark theme update to fix most of the issues seen in https://github.com/wallabag/wallabag/issues/4898

### CODE REVIEW

File: app/Resources/static/themes/material/css/dark_theme.scss [hunk 1]
```diff
@@ -71,8 +121,4 @@
   .collapsible {
     border-color: #222;
   }
-
-  table.striped > tbody > tr:nth-child(2n+1) {
-    background-color: #0c0c0c;
-  }
-}
+}
```
Comment (j0k3r) [inline app/Resources/static/themes/material/css/dark_theme.scss:124]: Put a new line at the end of the file

**Review by j0k3r — CHANGES_REQUESTED** [review]
You should update assets using `yarn build:prod` to fix the only red build.

File: app/Resources/static/themes/material/css/dark_theme.scss (hunk 1 above)
Comment (techexo) [reply app/Resources/static/themes/material/css/dark_theme.scss]: Commit abed407

**Comment by techexo** [comment]
@j0k3r I don't get what's going on at the moment. I added the EOF newline as requested and while it passes without issue `yarn build:prod` on my wallabag instance, the Building assets test doesn't pass on Github. Am I missing something here?

**Comment by j0k3r** [comment]
If the build assets fail, it means there are some difference between was is committed and what `yarn build:prod` is generating.
To fix that, you have to run `yarn build:prod` locally and commits generated assets files.

**Comment by techexo** [comment]
I did what was requested but I don't know how to reload tests on this PR to check if the assets are OK or not this time. I added the web/wallasets/ files that were generated by yarn build:prod to my commit.

**Comment by nicosomb** [comment]
@techexo could you run a `yarn build:prod` and commit the 2 changed files please?

**Comment by Kdecherf** [comment]
I've resolved the conflict and pushed the last version of compiled assets.

Here are two screenshots to compare the change (left is before, right is after):

2021-01-24-153623_2557x1308

2021-01-24-153634_2543x1299

**Review by Kdecherf — APPROVED** [review]

**Comment by Simounet** [comment]
I updated the code with a PR on his repository. https://github.com/techexo/wallabag/pull/1

**Comment by Kdecherf** [comment]
@Simounet woops, you may need to rebase your own PR now

**Comment by j0k3r** [comment]
Should we take care of that?

**Comment by Kdecherf** [comment]
@Simounet @j0k3r we could merge this PR as-is if it's ok for you, then @Simounet you could improve it via a new PR on this repo.

**Review by j0k3r — APPROVED** [review]
I agree @Kdecherf

**Pull request merged**